installation.md

```diff
@@ -72,6 +72,15 @@ ruby --version    # 2.7.2
 bundler --version # 2.2.0
 rails --version   # 6.1.0
 ```
-
+* We need `nodejs` package for a few of the JS gems. So you can install `nodejs` by running the following command:
```

Comment:
"JS gems" is a confusing statement - after all, gems are supposed to be in ruby. 

Let's simply say what the official installation guide says:

> Finally, you'll need Node.js and Yarn installed to manage your application's JavaScript.